WalletWasabi.Fluent/ViewModels/Wallets/Send/ChangeAvoidanceSuggestionViewModel.cs

```diff
@@ -50,7 +50,7 @@ public partial class ChangeAvoidanceSuggestionViewModel : SuggestionViewModel
 		[EnumeratorCancellation] CancellationToken cancellationToken)
 	{
 		var selections = ChangelessTransactionCoinSelector.GetAllStrategyResultsAsync(
-			transactionInfo.Coins,
+			transactionInfo.Coins.Where(c => c.IsPrivate(wallet.KeyManager.AnonScoreTarget)),
```

Comment:
the `TransactionInfo` already knows the `AnonScoreTarget` so, it could have a property called something like `PrivateCoins`.